src/patternfly/components/Page/page.scss

```diff
@@ -287,6 +297,29 @@
     }
   }
 
+  .pf-c-button.pf-m-opened {
```

Comment:
There's a noticeable jump when `.pf-m-opened` is applied to `.pf-c-button` because the widths are applied to the modifier rather than the base button. `.pf-m-opened` needs to apply to the default `.pf-c-button` to avoid shift.

Reply:
oh yes thanks for pointing that out!